packages/react/src/components/Animation/Animation.tsx

```diff
@@ -102,23 +134,65 @@ class Animation extends UIComponent<WithAsProp<AnimationProps>, any> {
     keyframeParams: PropTypes.object,
     playState: PropTypes.string,
     timingFunction: PropTypes.string,
+    visible: PropTypes.bool,
+    mountOnEnter: PropTypes.bool,
+    appear: PropTypes.bool,
+    unmountOnExit: PropTypes.bool,
+    timeout: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.shape({
+        appear: PropTypes.number,
+        enter: PropTypes.number,
+        exit: PropTypes.number,
+      }),
+    ]),
+  }
+
+  static defaultProps = {
+    timeout: 0,
```

Comment:
Is it a proper default?

Reply:
Yep, this is the default in Transition as well (`0` for both entering and exiting) and it basically means no removing/mounting, the element will always be in the dom.